packages/base/src/asset-registries/LocaleData.js

```diff
@@ -80,44 +64,50 @@ const getModuleContent = moduleName => {
 	throw new Error(`Unknown module ${moduleName}`);
 };
 
-const fetchCldr = async (language, region, script) => {
-	resolveMissingMappings();
-	const localeId = calcLocale(language, region, script);
+// load bundle over the network once
+const _loadCldrOnce = async localeId => {
+	const loadCldr = loaders.get(localeId);
 
-	let cldrObj = cldrData[localeId];
-	const url = cldrUrls[localeId];
-
-	const OpenUI5Support = getFeature("OpenUI5Support");
-	if (!cldrObj && OpenUI5Support) {
-		cldrObj = OpenUI5Support.getLocaleDataObject();
+	if (!cldrPromises.get(localeId)) {
+		cldrPromises.set(localeId, loadCldr(localeId));
 	}
 
-	if (cldrObj) {
-		// inlined from build or fetched independently
-		registerModuleContent(`sap/ui/core/cldr/${localeId}.json`, cldrObj);
-	} else if (url) {
-		// fetch it
-		const cldrContent = await fetchJsonOnce(getEffectiveAssetPath(url));
-		registerModuleContent(`sap/ui/core/cldr/${localeId}.json`, cldrContent);
-	}
+	return cldrPromises.get(localeId);
 };
 
-const registerCldr = (locale, url) => {
-	cldrUrls[locale] = url;
-};
+// external getAsync
+const fetchCldr = async (language, region, script) => {
+	const localeId = calcLocale(language, region, script);
 
-const setCldrData = (locale, data) => {
-	cldrData[locale] = data;
-};
+	// t o d o make loader
+	// const OpenUI5Support = getFeature("OpenUI5Support");
+	// if (!cldrObj && OpenUI5Support) {
+	// 	cldrObj = OpenUI5Support.getLocaleDataObject();
+	// }
 
-const getCldrData = locale => {
-	return cldrData[locale];
+	// fetch it
+	try {
+		const cldrContent = await _loadCldrOnce(localeId);
+		registerModuleContent(`sap/ui/core/cldr/${localeId}.json`, cldrContent);
+	} catch (e) {
+		if (!reportedErrors.has(e.message)) {
+			reportedErrors.add(e.message);
+			console.error(e.message); /* eslint-disable-line */
+		}
+	}
 };
 
-const _registerMappingFunction = mappingFn => {
-	cldrMappingFn = mappingFn;
+const registerLocaleDataLoader = (localeId, loader) => {
+	loaders.set(localeId, loader);
 };
 
+// register default loader from ui5 CDN
+SUPPORTED_LOCALES.forEach(localeId => {
```

Comment:
This is for the default locales when assets are not configured (dev workflow). So putting it in `Assets.js` (which is not imported anywhere by default) will break the default `ui5-datepicker` usage for example.

As discussed, I am switching this to "en" only so that developers can more easily see that locale assets still need to be configured.